src/app/components/toast/index.html

```diff
@@ -0,0 +1,92 @@
+<sky-demo-page
+  pageTitle="Toast"
+  skyuxModule="SkyToastModule"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The toast service launches a container over a page's content to display small messages. The <stache-code>SkyToastService</stache-code> provider launches toast messages and provides a common look-and-feel. The <stache-code>SkyToastMessage</stache-code> class watches for and triggers toast events. It also allows you to close messages.
+    </p>
+    <sky-alert
+      alertType="info">
+      The <stache-code>SkyToastService</stache-code> dynamically generates the toast component and appends it directly to the document's <stache-code>body</stache-code> element.
+    </sky-alert>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties sectionHeading="Flyout service methods">
+    <sky-demo-page-property
+      propertyName="open(config)">
+      Opens a message and displays the configured content. The <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code>. This method returns the SkyToastMessage instance.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="openMesage(message, config?)">
+      Opens a message toast and displays the provided message. The <stache-code>message</stache-code> parameter is of type <stache-code>string</stache-code> and determines the message to be displayed. The optional <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code> and adds additional configuration options to the toast. This method returns the SkyToastMessage instance.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="openTemplatedMessage(customComponentType, config?, providers?)">
+      Opens a custom toast and renders the provided component in a toast. The <stache-code>customComponentType</stache-code> parameter is of type <stache-code>Type&lt;SkyToastCustomComponent&gt;</SkyToastCustomComponent></stache-code> and determines the component to be rendered. The optional <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code> and adds additional configuration options to the toast. The optional <stache-code>providers</stache-code> parameter is of type <stache-code>Provider[]</stache-code> and specifies an array of custom providers to pass to the <stache-code>customComponentType</stache-code>'s constructor. This method returns the SkyToastMessage instance.
```

Comment:
Assuming my edits on the previous 2 first sentences are good, I'd change this one to "Displays a custom toast component and its message." ... But let me know if I'm just not following how these properties work and I can work on this some more.

In the second sentence, we don't need the closing tag within the code snippet. Also, I'd change "to be rendered" to "to render." ... And do we need a "SkyToastCustomComponent" section like the "SkyToastConfig" section?

In the third sentence, I'd delete "additional." ... In the "Provider" code snippet, do we want to include the brackets? I don't think we usually do this, but I'm not sure.

And in the last sentence, "SkyToastMessage" should be a code snippet.